packages/extensions/openapi-to-cadl/test/anomalyDetector/cadl-output/models.tsp

```diff
@@ -1,81 +1,84 @@
-import "@cadl-lang/rest";
-import "@azure-tools/cadl-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-core";
```

Comment:
This got added twice, probably a bug somewhere

Reply:
Good catch. I'll check whats going on